exporter/datadogexporter/factory.go

```diff
@@ -86,6 +91,31 @@ func createMetricsExporter(
 		return nil, err
 	}
 
+	go func() {
+		// Send host metadata
```

Comment:
Could be done separately/tracked as an improvement, but ideally this should also send a host metadata payload every 30 minutes, so that:

* Datadog continues to see current host metadata payloads from this collector (even if, based on the current code, it looks like the payload's contents would not change as long as the collector isn't restarted?)
* if the payload fails to be sent, this is eventually resolved with the next payload 30 minutes later

Reply:
I will do this separately, right now it doesn't change but I want to add support for EC2 tags so this makes sense. Thanks!